include/openssl/core_names.h

```diff
@@ -432,7 +432,7 @@ extern "C" {
 #define OSSL_EXCHANGE_PARAM_KDF_DIGEST            "kdf-digest" /* utf8_string */
 #define OSSL_EXCHANGE_PARAM_KDF_DIGEST_PROPS      "kdf-digest-props" /* utf8_string */
 #define OSSL_EXCHANGE_PARAM_KDF_OUTLEN            "kdf-outlen" /* size_t */
-/* The following parameter is octet_string on set and octet_ptr on get */
+/* The following parameter is an octet_string on set and octet_ptr on get */
```

Comment:
Strictly there should also be an _an_ before the `octet_ptr` as well.

Reply:
Fixed and rebased as there was another conflict with master now in CHANGES.md.